.github/actions/test-templates/action.yaml

```diff
@@ -4,38 +4,38 @@ runs:
   using: "composite"
   steps:
   - name: 🧪 Test IceRpc + Protobuf Client Template
-    run: mkdir -p templates-test/protobuf-client && cd templates-test/protobuf-client && dotnet new icerpc-protobuf-client && dotnet build
+    run: mkdir -p templates-test/ProtobufClient && cd templates-test/ProtobufClient && dotnet new icerpc-protobuf-client && dotnet build /p:TreatWarningsAsErrors=true
```

Comment:
I changed the names because the project and class names are by default derived from the directory name, which will result in warnings from the analyzer.